crates/router/src/configs/settings.rs

```diff
@@ -941,6 +945,21 @@ pub struct ServerTls {
     pub certificate: PathBuf,
 }
 
+#[cfg(feature = "v2")]
+#[derive(Debug, Clone, Deserialize, PartialEq, Eq)]
+pub struct CellInformation {
+    pub id: common_utils::id_type::CellId,
+}
+
+#[cfg(feature = "v2")]
+impl Default for CellInformation {
+    fn default() -> Self {
+        #[allow(clippy::expect_used)]
+        let cell_id = common_utils::id_type::CellId::from_str("default_cell_id") // This is to provide a default cell id for application settings.
+            .expect("Failed to create a default for Cell Id"); // Ideally we're not meant to have a default for Cell Id, so a panic here is acceptable here.
+        Self { id: cell_id }
```

Comment:
Nit:

```suggestion
        // We provide a static default cell id for constructing application settings.
        // This will only panic at application startup if we're unable to construct the default,
        // around the time of deserializing application settings.
        // And a panic at application startup is considered acceptable.
        #[allow(clippy::expect_used)]
        let cell_id = common_utils::id_type::CellId::from_str("default_cell_id") 
            .expect("Failed to create a default for Cell Id");
        Self { id: cell_id }
```